cypress/integration/ui_spec.js

```diff
@@ -0,0 +1,102 @@
+const {
+  user: { email, password },
+  apiKey,
+  env,
+  ...rest
+} = Cypress.env()
+
+const { host, adminUrl } = rest[env]
```

Comment:
Why do you do `rest[env]`, when above you're destructuring `env` ? Isn't it the same `env` ?
I think you don't need the `...rest` here :)

Reply:
It is not the same env!

`env` comes from the cmd itself example: `cypress open --env env=watch`
then I need `env` to fetch the correct credentials for the tests: 

```json
{
  "user": { ...  },
  "apiKey": "masterKey",
  "test": {
    "adminUrl": "http://localhost:1337/admin/auth/login",
    "host": "http://localhost:7700"
  },
  "watch": {
    "adminUrl": "http://localhost:8000/admin/auth/login",
    "host": "http://localhost:7700"
  },
  "ci": {
    "adminUrl": "http://localhost:1337/admin/auth/login",
    "host": "http://meilisearch:7700"
  }
}
```

The only way it would work is by doing this: 
```js
const {
  user: { email, password },
  apiKey,
  env,
  [env]: { host, adminUrl }
} = Cypress.env()
```

Which does not work.

Reply:
It is not the same env!

`env` comes from the cmd itself example: `cypress open --env env=watch`
then I need `env` to fetch the correct credentials for the tests: 

```json
{
  "user": { ...  },
  "apiKey": "masterKey",
  "test": {
    "adminUrl": "http://localhost:1337/admin/auth/login",
    "host": "http://localhost:7700"
  },
  "watch": {
    "adminUrl": "http://localhost:8000/admin/auth/login",
    "host": "http://localhost:7700"
  },
  "ci": {
    "adminUrl": "http://localhost:1337/admin/auth/login",
    "host": "http://meilisearch:7700"
  }
}
```

The only way it would work is by doing this: 
```js
const {
  user: { email, password },
  apiKey,
  env,
  [env]: { host, adminUrl }
} = Cypress.env()
```

Which does not work.

Reply:
Ok I got it ! Thanks for the explanations ! ❤️

Reply:
They are not the same env but I improved the deconstruction!